server/src/internalClusterTest/java/org/opensearch/search/basic/SearchWhileRelocatingIT.java

```diff
@@ -109,6 +109,7 @@ private void testSearchAndRelocateConcurrently(final int numberOfReplicas) throw
                     )
             );
         }
+        indexRandomForConcurrentSearch("test");
         indexRandom(true, indexBuilders.toArray(new IndexRequestBuilder[0]));
```

Comment:
Is this actually needed? The `indexRandom()` on L113 here should also be indexing dummy docs I believe.
If it *is* needed `indexRandomForConcurrentSearch` should come after `indexRandom` since at L112 no docs have been indexed into `"test"`.

Reply:
I didn't test it without adding it, but you're right it does work without it.  Will update the commit for the PR